eox_nelp/init_pipeline.py

```diff
@@ -82,3 +83,14 @@ def update_permissions():
         | rules.HasAccessRule("staff")
         | rules.HasAccessRule("instructor")
     )
+
+
+def patch_generate_password():
+    """This method patch `generate_password` of edx_django_util package,
+    with custom nelp `generate_password`.
+    """
+    # pylint: disable=import-outside-toplevel, unused-import
+    from edx_django_utils import user
+
+    from eox_nelp.user_authn.utils import generate_password
+    user.generate_password = generate_password
```

Comment:
Thanks @johanseto for the fix today.

I believe we should patch the generate password locally inside the importing modules:

```
from lms.djangoapps.support.views import manage_user
manage_user.generate_password = generate_password
```

Otherwise the imports won't be updated in `manage_user` due to the patch being too late in the import order.

What do you think?

Reply:
Forking edx_django_utils is also an option.

Reply:
Thanks @OmarIthawi  the patching locally is managed this PR.
https://github.com/eduNEXT/eox-nelp/pull/208/files
Is doing what you recommend using eox-tenant and seems working. Let us check out the behavior.